stdlib/public/Concurrency/TaskStatus.cpp

```diff
@@ -543,17 +508,15 @@ SWIFT_CC(swift)
 static void swift_task_cancel_group_child_tasksImpl(TaskGroup *group) {
   // Acquire the status record lock.
   //
-  // We purposefully DO NOT make this a cancellation by itself.
-  // We are cancelling the task group, and all tasks it contains.
-  // We are NOT cancelling the entire parent task though.
+  // Guaranteed to be called from the context of the parent task that created
+  // the task group once we have rdar://86496341
```

Comment:
This is "once we have https://github.com/apple/swift/pull/40616 " which was merged already right?

// We try not to leave opaque radar links _if possible_ as it's hard for external developers to follow what we mean by it